identity-engine/embedded-auth-with-sdk/features/04_self_service_registration.feature

```diff
@@ -0,0 +1,94 @@
+@4 @no-ci
+Feature: 4.1 Self Service Registration with Email Activation and optional SMS
+
+  Background:
+    Given there is a new sign up user named Mary Acme
+  # Given a Profile Enrollment policy defined assigning new users to the Everyone Group and by collecting "First Name", "Last Name", and "Email", is allowed and assigned to a SPA, WEB APP or MOBILE application
```

Comment:
Should we remove the commented lines?

Reply:
I left those in from the copy and paste from the wiki for the public to see. But I can remove them and will.